safe_transaction_service/contracts/tasks.py

```diff
@@ -98,16 +99,22 @@ def reindex_contracts_without_metadata_task() -> int:
 
     :return: Number of contracts missing
     """
-    i = 0
+    batch_size = settings.REINDEX_CONTRACTS_METADATA_BATCH
+    countdown_increment = settings.REINDEX_CONTRACTS_METADATA_COUNTDOWN
+    total_addresses_reindexed = 0
+    countdown = 0
     for address in (
         Contract.objects.without_metadata().values_list("address", flat=True).iterator()
     ):
         logger.info("Reindexing contract %s", address)
         create_or_update_contract_with_metadata_task.apply_async(
-            (address,), priority=1
+            (address,), priority=1, countdown=countdown
         )  # Lowest priority
-        i += 1
-    return i
+        total_addresses_reindexed += 1
+        countdown += (
+            countdown_increment if total_addresses_reindexed % batch_size == 0 else 0
```

Comment:
If you reindex `50` addresses but the batch size is `100`, module is `50`, so you are increasing the countdown without reason

Reply:
It is increasing the coundown just when the module is equal to 0, so multiples of the `batch_size`.  
In your example, should be when `total_addresses_reindexed` is 100, 200, 300...